website/control/models.py

```diff
@@ -31,36 +32,37 @@
 
 
 
-
 # First, we want to register a signal. This page recommends putting this code
 # in models.py: http://docs.djangoproject.com/en/dev/topics/signals/
 
 # Called when new database connections are created (see below).
 def _prepare_newly_created_db_connection(sender, **kwargs):
-  from clearinghouse.common.api import maindb
-  maindb.init_maindb()
+    from clearinghouse.common.api import maindb
+    maindb.init_maindb()
 
 # If this is a modern-enough version of django to support specifying a function
 # to be called on database connection creation, then have it call init_maindb()
 # at that time. This is to help prevent init_maindb() from accidentally not
 # being called when it should be.
-if django.VERSION >= (1,1):
-  # connection_created only exists with django >= 1.1
-  import django.db.backends.signals
-  django.db.backends.signals.connection_created.connect(_prepare_newly_created_db_connection)
+if django.VERSION >= (1.1):
```

Comment:
@aaaaalbert pointed out that we can safely get rid of this version check. If someone uses django <= 1.1 we really can't help. Apart from this line I didn't spot any other exaggerated compatibility legacy.